### DISCUSSION TITLE
Centre align text in horizontal stack

### DISCUSSION THREAD

**Discussion by Iqbal198** [Clooos/Bubble-Card#853] [Questions about config, custom styles and templates] [answered]
Hi Ive fixed the width of the horizontal stack buttons but can’t find out how to centre align the text within the buttons. 

here the variables I’m using 
.bubble-background-color {
background-color: rgb(224,17,95) !important;
  border: 0px solid var(--primary-text-color);
  }
.bubble-name {
        font-size: 18px;
        font-weight: 600;
  }
.bubble-button {
  width: 150px
  }

![IMG_5370](https://github.com/user-attachments/assets/287dcf17-7bc6-4f13-9898-157207213a6e)

Can someone help me out with the correct variable please.

**Comment by maaso** [comment, marked as answer]
I see you've set a fixed width of 150px on all the buttons. Do you absolutely need them to be the same size?

If not, this should fix the alignment:

```
.bubble-background-color {
    background-color: rgb(224,17,95) !important;
    border: 0px solid var(--primary-text-color);
}
.bubble-name {
   font-size: 18px;
   font-weight: 600;
   margin: 0 !important;
}
```

if they do need to all be the same size, try setting the size on the `.bubble-name` class and adding `text-align: center`:

```
.bubble-background-color {
    background-color: rgb(224,17,95) !important;
    border: 0px solid var(--primary-text-color);
}
.bubble-name {
    font-size: 18px;
    font-weight: 600;
    margin: 0 !important;
}
.bubble-name {
    width: 150px;
    text-align: center;
}
```

**Comment by Iqbal198** [comment]
Thanks for replying.

you second suggestion did the job!


Sent from my iPhone

On 19 Oct 2024, at 16:25, Maarten Somers ***@***.***> wrote:

﻿

I see you've set a fixed width of 150px on all the buttons. Do you absolutely need them to be the same size?

If not, this should fix the alignment:

.bubble-background-color {
    background-color: rgb(224,17,95) !important;
    border: 0px solid var(--primary-text-color);
}
.bubble-name {
   font-size: 18px;
   font-weight: 600;
   margin: 0 !important;
}
